forum/qa-theme/SnowFlat/qa-theme.php

```diff
@@ -247,38 +247,30 @@ public function body_content()
         $this->widgets('full', 'top');
         $this->header();
 
-        $address = qa_path_absolute(qa_request());
+        $request = qa_request();
+        $divideRequest = explode('/', $request);
         $pages = [
-            'http://forum.pasja-informatyki.pl/',
-            'http://forum.pasja-informatyki.pl/users',
-            'http://forum.pasja-informatyki.pl/polls',
-            'http://forum.pasja-informatyki.pl/messages',
-            'http://forum.pasja-informatyki.pl/messages/sent',
-            'http://forum.pasja-informatyki.pl/updates',
-            'http://forum.pasja-informatyki.pl/questions',
-            'http://forum.pasja-informatyki.pl/questions?sort=hot',
-            'http://forum.pasja-informatyki.pl/questions?sort=votes',
-            'http://forum.pasja-informatyki.pl/questions?sort=answers',
-            'http://forum.pasja-informatyki.pl/questions?sort=views',
-            'http://forum.pasja-informatyki.pl/questions?sort=hot',
-            'http://forum.pasja-informatyki.pl/users/special',
-            'http://forum.pasja-informatyki.pl/users/blocked'
+            '',
+            'polls',
+            'updates',
+            'unanswered',
+            'questions',
         ];
-        if (in_array($address, $pages) || (substr($address, 0, 42) === 'http://forum.pasja-informatyki.pl/message/')) {
-            $usersPages = [
-                'http://forum.pasja-informatyki.pl/users',
-                'http://forum.pasja-informatyki.pl/users/special',
-                'http://forum.pasja-informatyki.pl/users/blocked'
-            ];
-            if (in_array($address,
-                    $usersPages) && qa_is_logged_in() && qa_get_logged_in_level() >= QA_USER_LEVEL_MODERATOR
-            ) {
-                $this->output('<div class="qa-body-wrapper" style="margin: -3px auto !important;">', '');
-            } elseif (!in_array($address, $usersPages)) {
-                $this->output('<div class="qa-body-wrapper" style="margin: -3px auto !important;">', '');
-            } else {
-                $this->output('<div class="qa-body-wrapper">', '');
-            }
+        $partPages = [
+            'messages',
+            'message'
+        ];
+        $removeMargin = false;
+        if (in_array($request, $pages) || in_array($divideRequest[0], $partPages)) {

```

Comment:
no to wrzuć w jednego ifa z odpowiednimi bracketami oraz formatowaniem i będzie git